pom.xml

```diff
@@ -258,7 +258,7 @@
     </scm>
 
     <properties>
-        <ballerina.lang.version>1.0.4-SNAPSHOT</ballerina.lang.version>
+        <ballerina.lang.version>1.0.4</ballerina.lang.version>
```

Comment:
@keizer619 shouldn't this be 1.0.5-SNAPSHOT? :)

Reply:
Its picking latest stable version as the dependency.